744_find_smallest_letter.py

```diff
@@ -0,0 +1,13 @@
+from typing import List
+
+
+class Solution:
+    def nextGreatestLetter(self, letters: List[str], target: str) -> str:
+        letters.sort()
```

Comment:
You don't have to sort the letters because it  has been already sorted according to the problem description.

Reply:
Sure, I delete its sentence.